src/frontend/Sider/Behandling/Totrinnskontroll/SendtTilBeslutter.tsx

```diff
@@ -5,7 +5,7 @@ import styled from 'styled-components';
 
 import { Alert, BodyShort, Button, Heading } from '@navikt/ds-react';
 
-import { TotrinnskontrollOpprettet } from './typer';
+import { TotrinnskontrollOpprettet, TotrinnskontrollResponse } from './typer';
```

Comment:
Lyst til å ha en samtale om navngivningen og bruken av `Totrinnskontroll` for `TotrinnskontrollOpprettet` 🤠 (ikke en del av denne PR-en da)